glossary.md

```diff
@@ -698,15 +710,24 @@ Game theory is the study of mathematical models of strategic interaction between
 ## generating
 ## generator
 ## genetic algorithms
+
+A **genetic algorithm** (GA) is a variant of stochastic beam search in which successor states are generated by combining two parent states rather than by modfying a single state.
```

Comment:
```suggestion
A **genetic algorithm** (GA) is a variant of stochastic beam search in which successor states are generated by combining two parent states rather than by modifying a single state.
```